docs/developer/architecture.rst

```diff
@@ -236,25 +236,21 @@ This is the list of execution frameworks and memory formats supported in Modin.
 would like to contribute a new execution framework or memory format, please see the
 documentation page on :doc:`../contributing`.
 
-- `Pandas on Ray`_
+- :doc:`Pandas on Ray </UsingPandasonRay/index>`
     - Uses the Ray_ execution framework.
     - The compute kernel/in-memory format is a pandas DataFrame.
-- `Pandas on Dask`_
+- :doc:`Pandas on Dask </UsingPandasonDask/index>`
     - Uses the `Dask Futures`_ execution framework.
     - The compute kernel/in-memory format is a pandas DataFrame.
-- `Pyarrow on Ray`_ (experimental)
+- :doc:`Pyarrow on Ray </UsingPyarrowonRay/index>` (experimental)
     - Uses the Ray_ execution framework.
     - The compute kernel/in-memory format is a pyarrow Table.
 
-.. _pandas Dataframe: https://pandas.pydata.org/pandas-docs/version/0.23.4/generated/pandas.DataFrame.html
+.. _pandas Dataframe: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html
 .. _Arrow tables: https://arrow.apache.org/docs/python/generated/pyarrow.Table.html
 .. _Ray: https://github.com/ray-project/ray
 .. _code: https://github.com/modin-project/modin/blob/master/modin/engines/base/frame/data.py
-.. _Contributing: contributing.html
```

Comment:
Please, fix `contributing` link above as for the rest (remove `..`).

Reply:
fixed